jbpm-services/jbpm-executor/src/test/java/org/jbpm/executor/BasicExecutorBaseTest.java

```diff
@@ -308,7 +311,11 @@ public void reoccurringExecutionTest() throws InterruptedException {
         List<RequestInfo> executedRequests = executorService.getCompletedRequests(new QueryContext());
         assertEquals(3, executedRequests.size());
 
-
+        RequestInfo firstRequest = executedRequests.get(0);
+        RequestInfo lastRequest = executedRequests.get(executedRequests.size()-1);
+        long diffInMillies = Math.abs(lastRequest.getTime().getTime() - firstRequest.getTime().getTime());
+        // last request should be 2x NextRun = 4s later than first request, regardless of the sleepTime
+        assertEquals(4, TimeUnit.SECONDS.convert(diffInMillies, TimeUnit.MILLISECONDS));
```

Comment:
I would not use equals here... as it would lead to probably to flacky  tests

Reply:
I have changed the test assertion to be in line with others in the same class.